packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -448,6 +453,17 @@ class Auth {
         if (accessToken && !this.isTokenExpired(accessToken)) {
             return this.data
```

Comment:
Should dw_dnt be included in the this.data?

Reply:
Good question. @jeremy-jung1 do you know if the `dw_dnt` cookie should be returned as an attribute of `TokenResponse` ? In the API guides ([here](https://developer.salesforce.com/docs/commerce/commerce-api/references/shopper-login?meta=getAccessToken)) there is no `dw_dnt` in the response body, not sure if we want to add it.

Reply:
`dw_dnt` is a separate cookie that is not in the token response. `dnt` is in the access_token, but we don't have a use for that being in `this.data`